concrete/src/Validator/ValidatorManagerInterface.php

```diff
@@ -1,23 +1,21 @@
 <?php
+
 namespace Concrete\Core\Validator;
 
 /**
- * Interface ValidatorManagerInterface
  * A generic validator manager interface that enables validating against many validators at once.
- *
- * \@package Concrete\Core\Validator
  */
 interface ValidatorManagerInterface extends ValidatorInterface
 {
     /**
-     * Get a list of all validators.
+     * Get a list of all validators keyed by their handles.
      *
-     * @return ValidatorInterface[] Array of validators keyed by their handles
+     * @return \Concrete\Core\Validator\ValidatorInterface[]
```

Comment:
Providing the full namespace here doesn't really do anything. These hints are effected by our namespace declaration on ln 3

Reply:
Our ide symbols generator is not smart enough to create phpdoc resolving classes in the `use` statements, that's why I now always write the fully-qualified class names in phpdoc.

Reply:
That makes sense. We should fix this.